frontend/src/lib/utils.tsx

```diff
@@ -1280,3 +1281,10 @@ export function isGroupType(actor: ActorType): actor is GroupActorType {
 export function mapRange(value: number, x1: number, y1: number, x2: number, y2: number): number {
     return Math.floor(((value - x1) * (y2 - x2)) / (y1 - x1) + x2)
 }
+
+export function getEventNamesForAction(actionId: string | number, allActions: ActionType[]): string[] {
+    const id = parseInt(String(actionId))
```

Comment:
We can't be sure what comes in through a filter. Both options are allowed, and I need this typecast just to keep TS happy. Otherwise I'd have to force a type, and potentially make a mistake in the process (as in it's actually the wrong one in reality)